Lineup.cpp

```diff
@@ -0,0 +1,45 @@
+#include "Lineup.h"
+
+
+Lineup::Lineup()
+{
+    std::cout << "a new Lineup was created" << std::endl;
+}
+
+Lineup::~Lineup()
+{
+    std::cout << "a Lineup was destroyed" << std::endl;
+    tuesday.isBusy = false;
+}
+
+void Lineup::getSurferInfo( SurfReport::Surfer surfer ) 
+{
+    if ( surfer.atBeach == true)
```

Comment:
the `== true` is not needed because the member variable being compared is a `bool`.
- [ ] revise